NEWS.rst

```diff
@@ -1,3 +1,12 @@
+In development
+==============
+
+Internal changes
+----------------
+
+* #27687: The various org funk relations now accept a ``user_key`` parameter.
```

Comment:
This should be a new feature, shouldn't it?

Reply:
I guess it is, yes